tox.ini

```diff
@@ -42,7 +42,7 @@ deps =
 extras =
     test
 commands =
-    pytest --pyargs stixcore {toxinidir}/docs --cov stixcore --cov-config={toxinidir}/setup.cfg --remote-data=any -m "not end2end" {posargs}
+    pytest --pyargs stixcore {toxinidir}/docs --cov stixcore --cov-config={toxinidir}/setup.cfg --remote-data=any -m "not end2end" -n 1 --dist loadfile {posargs}
```

Comment:
Is the distributed test part of this PR?

Reply:
the preparations somehow yes 
but n is set to 1 so ineffective at the moment